server/app/services/program/ProgramService.java

```diff
@@ -346,6 +346,30 @@ ProgramDefinition setProgramQuestionDefinitionOptionality(
       throws ProgramNotFoundException, ProgramBlockDefinitionNotFoundException,
           ProgramQuestionDefinitionNotFoundException;
 
+  /**
+   * Set a program question definition to enable address correction. If the question definition ID
+   * is not present in the program's block, then nothing is changed.
```

Comment:
question (blocking):

> If the question definition ID is not present in the program's block, then nothing is changed.

This makes it sounds like it fails silently, in contrast to an exception being thrown.

Reply:
Removed that sentence.